app/views/publishers/vacancies/build/_steps.html.slim

```diff
@@ -0,0 +1,10 @@
+- if current_organisation.is_a?(School)
```

Comment:
Is the only difference between these two branches `school_vacancy_steps` vs `school_group_vacancy_steps`? It seems like too much repetition.